manageiq-operator/config/samples/_v1alpha1_manageiq.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: manageiq.org/v1alpha1
 kind: ManageIQ
 metadata:
-  name: manageiq-sample
+  name: miq
```

Comment:
What was wrong with manageiq-sample?

Reply:
I just overlayed our old one on top.  Switched to the default name